source/writing.html.erb.md

```diff
@@ -186,7 +186,7 @@ Use short headings to group related paragraphs and clearly describe the sections
 {:.attach_permalink}
 ## Make link text meaningful
 
-Write link text so that it clearly describes the content of the link target. Avoid using meaningless link text such as 'click here', 'find out more', or 'read more'. Indicate the document type and approximate size in the link text, when relevant, for example (RTF, 20mb). 
+Write link text so that it clearly describes the content of the link target by itself (without the surrounding text). Avoid using meaningless link text such as 'click here' or 'read more'. Indicate the document type and approximate size in the link text, when relevant, for example (RTF, 20mb). 
```

Comment:
suggestion: "Write link texts that it describe the content of the link target. Avoid using ambiguous link texts, such as ..." [...] "Also indicate relevant information about the link target, such as the document type and size when linking to documents."